### PR TITLE
Added fix and format flags

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Also addresses https://github.com/blackbaud/skyux2-docs/issues/322.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
This functionality has been released in [3.5.0](https://github.com/blackbaud/skyux-sdk-builder/blob/master/CHANGELOG.md#350-2019-03-08) and is ready for documentation.

File: src/stache/includes/cli/lint-fix.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<stache-code>--fix</stache-code> — Attempts to automatically "fix" any linting errors found.  The default is false.  Please use caution when setting this to true, as it will overwrite files. This flag is passed directly <stache-code>tslint</stache-code> library.
```
Comment (Blackbaud-DeniseCisneros) [inline src/stache/includes/cli/lint-fix.html]: What does "fix" mean? If it's really fixing it, I vote to remove the quotation marks so it's clearer. Also, is this a property or option? Wasn't sure how to reference it in the edits below.  

Attempts to automatically fix linting errors. By default, this property (option?) is set to <stache-code>false</stache-code>. Use caution when setting the value to true as it overwrites files. This flag is passed directly <stache-code>tslint</stache-code> library.

File: src/stache/includes/cli/lint-format.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<stache-code>--format [string]</stache-code> — Sets the tslint format option. This flag is passed directly <stache-code>tslint</stache-code> library.  See <a href="https://palantir.github.io/tslint/formatters/">their documentation</a> for a list of formatters.
```
Comment (Blackbaud-DeniseCisneros) [inline src/stache/includes/cli/lint-format.html]: Should tslint format be `tslint format`? Also, the second sentence is missing "to the" as in "This flag is passed directly to the `tslint` library."

File: src/stache/includes/cli/lint-fix.html (hunk 1 above)
Comment (Blackbaud-BobbyEarl) [reply src/stache/includes/cli/lint-fix.html]: Very good points.  @blackbaud-johnly had some very similar verbal feedback.  I believe he's going to submit a PR in to mine so we can make sure to get that addressed.

Comment (blackbaud-johnly) [reply src/stache/includes/cli/lint-fix.html]: Addressed in separate PR

File: src/stache/includes/cli/lint-format.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/stache/includes/cli/lint-format.html]: Addressed in separate PR

**Pull request merged**